static/app/views/alerts/rules/issue/ruleNode.tsx

```diff
@@ -335,7 +335,7 @@ function RuleNode({
               "This project doesn't support sessions. [link:View supported platforms]",
               {
                 link: (
-                  <ExternalLink href="https://docs.sentry.io/product/releases/health/setup/" />
+                  <ExternalLink href="https://docs.sentry.io/product/releases/setup/#release-health" />
```

Comment:
I'm looking at the text for the link `View supported platforms` and, to me, it seems the `release/health/setup` is more appropriate?